src/lib/child_processes/termination.ml

```diff
@@ -24,6 +24,17 @@ let mark_termination_as_expected t child_pid =
 
 let remove : t -> Pid.t -> unit = Pid.Table.remove
 
+(* for some signals that cause termination, offer a possible explanation *)
+let get_signal_cause_opt =
+  let open Signal in
+  let signal_causes_tbl : string Table.t = Table.create () in
+  List.iter
+    [ (kill, "Process likely killed because out of memory")
```

Comment:
You can remove `likely` here, since the wrapping error message already says `possible`